Source/ProjectRimFactory/Storage/UI/Dialog_OutputMinMax.cs

```diff
@@ -70,6 +70,10 @@ public override void DoWindowContents(Rect rect)
                 Text.Anchor = anchorBuffer;
                 Widgets.TextFieldNumeric(rectRight, ref outputSettings.min, ref minBufferString, 0);
             }
+            if (outputSettings.max < outputSettings.min)
+            {
+                maxBufferString = outputSettings.min.ToString();

```

Comment:
this seems odd to me. 
doesn't a updated of the value work?
if we need to update the string i would reuse the already existing string instead of creating a new one

Reply:
I noticed this is an interesting quirk of the Widgets.TextFieldNumeric function, when you change the string buffer, the linked int value would automatically get updated, too, so there'd be no need to manually update outputSettings.min or outputSettings.max .
However the inverse is not true, if you only update outputSettings.min or outputSettings.max , the corresponding string buffer won't update with it.

Thus I only wrote to update the string buffer.